src/mongo_client.ts

```diff
@@ -262,6 +267,16 @@ export interface MongoClientOptions extends BSONSerializeOptions, SupportedNodeC
    * TODO: NODE-5671 - remove internal flag
    */
   mongodbLogPath?: 'stderr' | 'stdout' | MongoDBLogWritable;
+  /**
+   * @internal
+   * TODO: NODE-5671 - remove internal flag
+   */
+  mongodbLogComponentSeverities?: LogComponentSeveritiesClientOptions;
+  /**
+   * @internal
+   * TODO: NODE-5671 - remove internal flag
+   */
+  mongodbLogMaxDocumentLength?: string;
```

Comment:
Why have this set as a string instead of a number?

Reply:
It's set as string in the environment options interface [`MongoLoggerEnvOptions`](https://github.com/mongodb/node-mongodb-native/blob/fd58eec441d99493aff5c62bcc170c469efa679b/src/mongo_logger.ts#L124):
```MONGODB_LOG_MAX_DOCUMENT_LENGTH?: string;```